mlops/evaluation/data/make_queries_from_index.ipynb

```diff
@@ -175,7 +186,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.12.3"
+   "version": "3.11.7"
```

Comment:
any reason for changing python version, Was this intentional?

Reply:
Yes, it's 3.11 in workflows and dev container. we can migrate to 3.12, but it should be tested.